Mathlib/LinearAlgebra/Matrix/GeneralLinearGroup/Defs.lean

```diff
@@ -147,6 +151,44 @@ theorem map_comp_apply (f : T →+* R) (g : R →+* S) (x : GL n T) :
     (map g).comp (map f) x = map g (map f x) :=
   rfl
 
+variable (f : R →+* S)
+
+@[simp]
+lemma map_one : map f (1 : GL n R) = 1 := by
```

Comment:
I would make the `map_*` lemmas protected

Reply:
ok, I made the new `map_*` lemmas protected.